performance_indices.py

```diff
@@ -160,10 +166,14 @@ def main(args):
 
     # new bunch of functions to set grids, create correction command, masks and areas
     comp = face['model']['component']  # Get component for each domain
-    atminifile, oceinifile = getinifiles(face, diag)
-    cdop.set_gridfixes(atminifile, oceinifile, comp['atm'], comp['oce'])
+    atminifile, ocegridfile, oceareafile = getinifiles(face, diag)
+    cdop.set_gridfixes(atminifile, ocegridfile, oceareafile, comp['atm'], comp['oce'])
     cdop.make_atm_masks(atminifile, extra=f'-invertlat -remapcon2,{diag.resolution}')
```

Comment:
Of course having a invertlat here is terribly mode specific .... we will have to solve it sooner or later (but no need to do it now)

Reply:
I am aware, but it actually comes from the lat grids reference files which are decreasing (I guess because they come from old grads version). Usually all the lat grids are increasing, and when we interpolate on the target grid this is solved automatically by CDO. I will convert all the lats of the reference files and remove the invertlat call.

Reply:
In the last commit I reverted the lats of the reference files and removed the call to invertlat!